Orange/widgets/visualize/owmosaic.py

```diff
@@ -1005,11 +1009,14 @@ def DrawLegend(self, data, x0_x1, y0_y1):
     # ########################################
     # cell/example selection
     def sendSelectedData(self):
+        data = self.data
+        if any(attr.is_continuous for attr in data.domain):
```

Comment:
This fails if `self.data` is `None`. To replicate, start Orange with new settings and connect File to Mosaic. Mosaic will fail with 

      File "Orange/widgets/visualize/owmosaic.py", line 1014, in sendSelectedData
        if any(attr.is_continuous for attr in data.domain):
    AttributeError: 'NoneType' object has no attribute 'domain'